app/views/reservations/_main_page_nav_buttons.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="btn-toolbar">
```

Comment:
Where did these come from? Isn't this the old nav bar from the index view before #708? Why are we keeping this partial?